katas/move-zeros/move-zeros.js

```diff
@@ -1,5 +1,16 @@
-function moveZeros() {
-
+function move_zeros(arrNum, isRight = true){
+  const zeroArray = [];
```

Comment:
because zeroArray is constant link to created array, we will not change this link, we will add new elements into existed array